frontend/src/pages/RequestDetailsPage/RequestDetailsPageV2/RequestDetailsPageV2Content.tsx

```diff
@@ -166,9 +135,9 @@ export function RequestDetailsPageContentV2({
               "2xl:min-w-[320px]",
             )}
           >
-            <TraceDetailsTimeline waterfall={waterfall} />
+            <TraceDetailsTimeline waterfall={waterfall} className="lg:pt-0" />
           </ResizablePanel>
-          <ResizableHandle />
+          <ResizableHandle className="max-lg:hidden" />
```

Comment:
hide the handle on smaller screens otherwise it gives a weird left border looking thing in the UI